jsonpaths/com.google.analytics.ecommerce/item_1.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
@paulboocock I think you're better positioned to assess its importance. I have no idea where it is used.